dist/obs-bundled-gems.spec

```diff
@@ -142,8 +142,10 @@ find %{buildroot} -type f -print0 | xargs -0 grep -l /usr/bin/env | while read f
   chmod a-x $file
 done
 
-# Remove rpaths from files
-chrpath --delete %{buildroot}%_libdir/obs-api/ruby/*/gems/sassc-*-x86_64-linux/lib/sassc/libsass.so
+# Remove rpaths from files (legacy)
+# if service bundle_gems ran with force_ruby_platform
+# the "|| true" prevents package build from breaking
+chrpath --delete %{buildroot}%_libdir/obs-api/ruby/*/gems/sassc-*-x86_64-linux/lib/sassc/libsass.so || true
```

Comment:
Now that the bundle_gems service run always with `force_ruby_platform true` maybe we can get rid of all the `chrpath` line. Can we try removing the `chrpath` line?

Reply:
sounds good to me!

Reply:
I`m ok with that if you are